cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -1640,8 +1640,11 @@ predicate localInstructionFlow(Instruction e1, Instruction e2) {
   localFlow(instructionNode(e1), instructionNode(e2))
 }
 
+/**
+ * INTERNAL: Do not use.
+ */
 cached
-private module ExprFlowCached {
+module ExprFlowCached {
```

Comment:
Should we add a brief note explaining why this isn't private?

Reply:
(so we know we can undo this once we've gotten rid of default taint tracking)

Reply:
Good idea. Fixed in 384ca0c31f9a8ae0c7d3d91afac8e242beffabdd.